src/i2c.rs

```diff
@@ -66,14 +66,6 @@ impl Mode {
             frequency: frequency.into(),
         }
     }
-
-    pub fn get_frequency(&self) -> Hertz {
-        match self {
-            &Mode::Standard { frequency } => frequency,
-            &Mode::Fast { frequency } => frequency,
-            &Mode::FastPlus { frequency } => frequency,
-        }
-    }
```

Comment:
I'm assuming this was removed, because a frequency is not available when using `Mode::Custom`. I'm not sure if we need it. If not, removing it is probably the best option.

If we do need it, it might be better to return `Option<Hertz>` instead.

Reply:
I feel like it is unnecessary, making things complicated. Frequency is a tuple struct so if anyone wants they can use .0 for that

Reply:
I feel like it is unnecessary, making things complicated. Frequency is a tuple struct so if anyone wants they can use .0 for that